websites/V/Viu/presence.ts

```diff
@@ -90,7 +90,7 @@ async function generateToken() {
 	return response.json().then(x => x.token);
 }
 
-/* eslint-disable camelcase */
```

Comment:
@Bas950 this should not be done.

Reply:
Hmm we should check why it even removes it

Reply:
![image](https://user-images.githubusercontent.com/32113157/235819729-043f8578-4bb3-4e62-9a69-595e796a7f64.png)

After removing the comment, eslint does not actually throw any warnings.

Reply:
It looks like eslint ignores camelcase 'issues' in interfaces.

Reply:
so then it can be removed?

Reply:
Yes

Reply:
> ![image](https://user-images.githubusercontent.com/32113157/235819729-043f8578-4bb3-4e62-9a69-595e796a7f64.png)
> 
> After removing the comment, eslint does not actually throw any warnings.

I tested it with #7320 it gives deepscan errors

Reply:
I added it because of Deepscan